Lib/asyncio/test_utils.py

```diff
@@ -361,6 +361,13 @@ def assert_writer(self, fd, callback, *args):
             handle._args, args)
 
     def _ensure_fd_no_transport(self, fd):
+        if not isinstance(fd, int):
+            try:
+                fd = int(fd.fileno())
+            except (AttributeError, TypeError, ValueError):
+                # This code matches `selectors._fileobj_to_fd` function.
+                raise ValueError("Invalid file object: "
```

Comment:
Not covered.

Reply:
This won't be covered as it's test_utils' test loop. There's no point in covering it.

Reply:
Maybe let's add `# pragma: no cover` comment for `except` clause?
It increases formal coverage level at least (: